.github/workflows/start-mysql.sh

```diff
@@ -18,7 +18,7 @@ wait_for_mysql() {
   port=$1
   echo "Waiting for MySQL at port $port..."
   attempts=0
-  while ! nc -w 1 localhost $port | grep -q "mysql"; do
+  while nc -vvw 1 localhost $port | grep -q "succeeded"; do
```

Comment:
mysql 8 has a differently formatted response (at least when using the `mysql/mysql-server` version

Reply:
This has been cleaned up now.